pinetree.js

```diff
@@ -0,0 +1,40 @@
+function tree(height, character) {
+  var growtree = [];
+  growtree[height] = character;
+
+  for (var i = 0; i < growtree.length; i++) {
+   growtree[i] = "" + " ";
+  }
+
+  for (var j = 0; j < height; j++) {
+    growtree[height - j] = " " + character + "";
+
+    var treeOutput = growtree.join("");
+    console.log(treeOutput);
+  }
+};
+
+var nameInput = document.getElementById("height");
+var btn = document.getElementById("btn");
+btn.addEventListener("click", validation);
+
+function validation(){
+var heightbtn = document.getElementById("height").value;
+var shapebtn = document.getElementById("shape").value;
+if (heightbtn === "" && shapebtn === "") {
```

Comment:
&& (and) needs to be || (or). The and only fires when BOTH are empty. But we need to alert the user when either of them are empty.